src/current/v25.2/vector-indexes.md

```diff
@@ -0,0 +1,323 @@
+~~~
+
+To enable the creation of vector indexes on non-empty tables, also disable the `sql_safe_updates` [session setting]({% link {{ page.version.version }}/session-variables.md %}#sql-safe-updates). This allows vector indexes to be backfilled on existing rows, during which **table writes are blocked** to ensure vector index consistency.
+
+{{site.data.alerts.callout_danger}}
+Adding a vector index to a non-empty table can temporarily disrupt workloads that perform continuous writes.
+{{site.data.alerts.end}}
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET sql_safe_updates = false;
+~~~
+
+## Create vector indexes
+
+To create a vector index, use the [`CREATE VECTOR INDEX`]({% link {{ page.version.version }}/create-index.md %}) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE VECTOR INDEX ON {table} (column});
+~~~
+
+You can also specify a vector index during table creation. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TABLE items (
+    category STRING,
+    embedding VECTOR(1536),
+    VECTOR INDEX (embedding)
+);
+~~~
+
+You can create a vector index with a *prefix column* to pre-filter the search space. This is especially useful for tables containing millions of vectors or more. For an example, refer to [Create and query a vector index](#create-and-query-a-vector-index).
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TABLE items (
+    category STRING,
+    embedding VECTOR(1536),
+    VECTOR INDEX (category, embedding)
+);
+~~~
+
+You can optionally specify an opclass. If not specified, the default is `vector_l2_ops`:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TABLE items (
+    category STRING,
+    embedding VECTOR(1536),
+    VECTOR INDEX embed_idx (embedding vector_l2_ops)
+);
+~~~
+
+## Comparisons
+
+Vector indexes on `VECTOR` columns support the following comparison operator:
+
+- **L2 distance**: [`<->`]({% link {{ page.version.version }}/functions-and-operators.md %}#operators)
+
+For usage, refer to the [Example](#example).
+
+## Tune vector indexes
+
+The following vector index parameters are tunable:
+
+- Search beam size, using `vector_search_beam_size`
+- Partition size, using `min_partition_size` and `max_partition_size`
+- Build beam size, using `build_beam_size` (not recommended)
+
+For guidelines on how these parameters affect search accuracy and write performance, refer to [Tuning considerations](#tuning-considerations).
+
+Set the following storage parameters when you create a vector index:
+
+- `min_partition_size`: Minimum number of vectors in a partition before it is merged with another partition. This value must be greater than or equal to `1`. The default value is `16`.
+- `max_partition_size`: Maximum number of vectors in a partition before it is split into smaller partitions. This must be at least 4 times the value of `min_partition_size`. The default value is `128`.
```

Comment:
The max value for `max_partition_size` is 4096, and the max value for `min_partition_size` is 1024.